### PR TITLE
Use Travis CI for Windows test

### PR DESCRIPTION
**Pull request by eed3si9n**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ install:
   - unset JAVA_HOME
   - java -Xmx32m -version
   - javac -J-Xmx32m -version
+  - sdk install sbt
```
Comment (lrytz) [inline .travis.yml]: Seems that broke the linux build...

```
Error: Could not find or load main class @.etc.sbt.sbtopts
```

**Review by lrytz — COMMENTED** [review]
Cool, that's very useful!


On Windows:
```
Looking for unzip...
Looking for zip...
Not found.
Please install zip on your system using your favourite package manager.
```

**Comment by dwijnand** [comment]
Make sure to inspect the speed of the Windows jobs in Travis CI.  Last time I tried using them they were significantly slower than Linux/Mac.

**Comment by eed3si9n** [comment]
> Looking for unzip...
> Looking for zip...

I might have to switch to jabba for Windows.

**Comment by lrytz** [comment]
According to https://docs.travis-ci.com/user/reference/windows, travis is running Git bash, and 7zip is installed. According to https://stackoverflow.com/a/49285294/248998, an `ln -s` to alias `7z.exe` might work?

**Comment by eed3si9n** [comment]
@dwijnand 

> Make sure to inspect the speed of the Windows jobs in Travis CI. Last time I tried using them they were significantly slower than Linux/Mac.

It was discussed in the meeting this week to look into it because it might be less pain compared to maintaining Jenkins Windows image.

**Comment by eed3si9n** [comment]
I needed both zip and unzip so I came up with the following after some trial and errors:

```
          - choco install zip
          - choco install unzip
```

**Comment by eed3si9n** [comment]
@dwijnand is right about Windows image being slow. "The job exceeded the maximum time limit for jobs" (90 min) at

```
# starting 1625 tests in pos
```

**Comment by dwijnand** [comment]
Sorry, I forgot to mention: my point was use AppVeyor (or possibly others) in alternative to Travis CI's slow instances.

**Pull request closed without merging**